apps/studio.giselles.ai/app/(auth)/components/action-prompt.tsx

```diff
@@ -10,7 +10,7 @@ export const ActionPrompt: FC<ActionPromptProps> = ({
 	action,
 	leftIcon,
 }) => (
-	<div className="text-[14px] text-black-30 font-rosart flex gap-[2px] items-center">
+	<div className="text-[14px] text-black-30 font-geist flex gap-[2px] items-center">
```

Comment:
Now that PR #597 (setting the default font to Geist) has been merged, this definition can be removed.
```suggestion
	<div className="text-[14px] text-black-30  flex gap-[2px] items-center">
```